node/pkg/proto/node/v1/node.pb.go

```diff
@@ -0,0 +1,1810 @@
+// Code generated by protoc-gen-go. DO NOT EDIT.
+// versions:
```

Comment:
Do we need to check in all the auto-generated go code?

Reply:
Yes, if other go projects need it, then we don't need to regenerate the code, wormhole also does this: https://github.com/wormhole-foundation/wormhole/tree/dev.v2/node/pkg/proto